test/test_support.ts

```diff
@@ -32,6 +35,8 @@ export const baseCompilerOptions: ts.CompilerOptions = {
   allowJs: false,
   importHelpers: true,
   noEmitHelpers: true,
+  baseUrl: '.',
+  paths: tslibPath ? {'tslib': [tslibPath]} : undefined,
```

Comment:
add a comment on the lookup mechanism for 'tslib'. It was a bit magic for me.

Reply:
Done.